### ISSUE TITLE
metainfo - "Image_name" in  Eval.py (Value Error)

### ISSUE THREAD

**Issue by Dipankar1997161** [david8862/tf-keras-simple-baselines-keypoint-detection#1]
Hey David,
**Could you tell me what is being passed on the "image_name" from the meta_info['name].**
I again got the error, I thought I resolved but it seems not. 
here is a snap of the error
![Screenshot (291)](https://user-images.githubusercontent.com/85514219/166090153-3f98da55-0328-4bb0-9b3b-51c9a2a67ae6.png)

**my images are named as for example " 0000115_img.png" Total 7 numbers followed by '_img' and then extension.**

It would be great if you could help here.

Questions:
Also, why is the Eval mode running after 1 epoch??  Can't the train.py complete its epochs first and then evaluate?

Is it possible to run eval.py without running the train.py successfully ?? This might be vital for me then to solve the error.

snap of the code:
![Screenshot (292)](https://user-images.githubusercontent.com/85514219/166090683-312b1fad-f235-4799-a7a3-60125942ef25.png)

**Comment by Dipankar1997161** [comment]
I have resolved the following issue based on my dataset! 
Thank you sir
